pjmedia/src/pjmedia-codec/vpx_packetizer.c

```diff
@@ -0,0 +1,215 @@
+/* 
+ * Copyright (C) 2020 Teluu Inc. (http://www.teluu.com)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+ */
+#include <pjmedia-codec/vpx_packetizer.h>
+#include <pjmedia/errno.h>
+#include <pjmedia/types.h>
+#include <pjmedia/vid_codec_util.h>
+#include <pj/assert.h>
+#include <pj/errno.h>
+#include <pj/log.h>
+#include <pj/pool.h>
+#include <pj/string.h>
+
+
+#if defined(PJMEDIA_HAS_VIDEO) && (PJMEDIA_HAS_VIDEO != 0)
+
+#define THIS_FILE		"vpx_packetizer.c"
+
+/* VPX packetizer definition */
+struct pjmedia_vpx_packetizer
+{
+    /* Current settings */
+    pjmedia_vpx_packetizer_cfg cfg;
+};
+
+/*
+ * Create vpx packetizer.
+ */
+PJ_DEF(pj_status_t) pjmedia_vpx_packetizer_create(
+				pj_pool_t *pool,
+				const pjmedia_vpx_packetizer_cfg *cfg,
+				pjmedia_vpx_packetizer **p)
+{
+    pjmedia_vpx_packetizer *p_;
+
+    PJ_ASSERT_RETURN(pool && p, PJ_EINVAL);
+
+    if (cfg && cfg->fmt_id != PJMEDIA_FORMAT_VP8 &&
+	cfg->fmt_id != PJMEDIA_FORMAT_VP9)
+    {
+	return PJ_ENOTSUP;
+    }
+
+    p_ = PJ_POOL_ZALLOC_T(pool, pjmedia_vpx_packetizer);
+    if (cfg) {
+	pj_memcpy(&p_->cfg, cfg, sizeof(*cfg));
+    } else {
+	p_->cfg.fmt_id = PJMEDIA_FORMAT_VP8;
```

Comment:
`mtu` should be initialized too?
even better if there is `pjmedia_vpx_packetizer_cfg_default()`.